app/models/user.rb

```diff
@@ -7,7 +7,9 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :trackable, :validatable
 
   belongs_to :draw
-  belongs_to :group
+  has_one :membership
```

Comment:
Yeah I think we deal with it later; shouldn't be too bad to change this to a `has_many :through` eventually